recipes/libcurl/all/test_package/CMakeLists.txt

```diff
@@ -4,5 +4,7 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+find_package(CURL)
+
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
\ No newline at end of file
+target_link_libraries(${PROJECT_NAME} CURL::CURL)
```

Comment:
According to CMake the target should be `CURL::libcurl`, but it is not doable with Conan right now and, IMO it is much better to ensure that CMake usesour `FindCURL.cmake`